### ISSUE TITLE
[RELEASE] 1.5.2

### ISSUE THREAD

**Issue by innobead** [longhorn/longhorn#6981]
**What's the task? Please describe.**
Action items for releasing v1.5.2

**Describe the sub-tasks.**
  - Pre-Release
    - [x] Regression test plan (manual) - @khushboo-rancher (https://github.com/longhorn/longhorn/wiki/Release-Test-Plan---1.5.2)
    - [x] Run e2e regression for pre-GA milestones (`install`, `upgrade`) - @yangchiu
    - [x] Run security testing of container images for pre-GA milestones - @roger-ryao 
    - [x] Verify longhorn chart PR to ensure all artifacts are ready for GA (`install`, `upgrade`)  @chriscchien 
    - [x] Run core testing (install, upgrade) for the GA build from the previous patch and the last patch of the previous feature release (1.5.1). - @yangchiu 
  - Release
    - [x] Release longhorn/chart from the release branch to publish to ArtifactHub
    - [x] Release note
	     - [x] Deprecation note
	     - [x] Upgrade notes including highlighted notes, deprecation, compatible changes, and others impacting the current users
  - Post-Release
    ~- [ ] Create a new release branch of manager/ui/tests/engine/longhorn instance-manager/share-manager/backing-image-manager when creating the RC1~
    - [ ] Update https://github.com/longhorn/longhorn/blob/master/deploy/upgrade_responder_server/chart-values.yaml @PhanLe1010 (reviewer: @mantissahz )
    - [ ] Add another request for the rancher charts for the next patch release (`1.5.3`) @rebeccazzzz  
  - Rancher charts: verify the chart is able to install & upgrade - @khushboo-rancher 
    - [ ] rancher/image-mirrors update @mantissahz  (reviewer: @PhanLe1010 )
    - [ ] rancher/charts 2.7 & 2.8 (for any active versions) branches for rancher marketplace @mantissahz  (reviewer: @PhanLe1010)
    
cc @longhorn/qa @longhorn/dev

**Comment by innobead** [comment]
Changelogs since v1.5.1.

#### longhorn-ui
- release: v1.5.2-rc1 [84a27a6](https://github.com/longhorn/longhorn-ui/commit/84a27a6)
- fix(volume) Add tooltip for volumes in maintenance mode [07f97cf](https://github.com/longhorn/longhorn-ui/commit/07f97cf)
- feat(volume) Add volume scheduling failure message [cce1b84](https://github.com/longhorn/longhorn-ui/commit/cce1b84)

#### longhorn-manager
- release: v1.5.2-rc1 [2d14e404](https://github.com/longhorn/longhorn-manager/commit/2d14e404)
- fix(backup): AWS IAM Role Annotation [cd5ebc54](https://github.com/longhorn/longhorn-manager/commit/cd5ebc54)
- fix(setting): put settings' ReadOnly into effect [ab42533e](https://github.com/longhorn/longhorn-manager/commit/ab42533e)
- Filter pdb, daemonset and deployment by namespace [9fa7892b](https://github.com/longhorn/longhorn-manager/commit/9fa7892b)
- Refactor informer factories creation and initialization [f291376b](https://github.com/longhorn/longhorn-manager/commit/f291376b)
- Update unit tests [650de9bc](https://github.com/longhorn/longhorn-manager/commit/650de9bc)
- Filter Kubernetes resources by longhorn-system namespace [16402f01](https://github.com/longhorn/longhorn-manager/commit/16402f01)
- Fix backport conflicts. [dc4ee128](https://github.com/longhorn/longhorn-manager/commit/dc4ee128)
- Update vendor after backupstore merge. [163e178d](https://github.com/longhorn/longhorn-manager/commit/163e178d)
- fix: add trace log to invalid state error [da6e549f](https://github.com/longhorn/longhorn-manager/commit/da6e549f)
- Suppress nagging error logging. Remove debug logging. Incorporate PR feedback. Change error name per review. [ee5fc324](https://github.com/longhorn/longhorn-manager/commit/ee5fc324)
- fix(storage-network): incompatible with Multus version above 4.0.0 [a67dc7fb](https://github.com/longhorn/longhorn-manager/commit/a67dc7fb)
- Revert vendor changes until later. [343e0eb1](https://github.com/longhorn/longhorn-manager/commit/343e0eb1)
- Tweak validation of backuptarget URL. [49192a96](https://github.com/longhorn/longhorn-manager/commit/49192a96)
- Eliminate multiple client creations [cb5572b8](https://github.com/longhorn/longhorn-manager/commit/cb5572b8)
- fix(import): typo [a5d7ca8b](https://github.com/longhorn/longhorn-manager/commit/a5d7ca8b)
- style: imports [b0d5a112](https://github.com/longhorn/longhorn-manager/commit/b0d5a112)
- feat(log): add function name and line number in log message [c75dac42](https://github.com/longhorn/longhorn-manager/commit/c75dac42)
- Delete dummy CSI services during upgrade [71e30a96](https://github.com/longhorn/longhorn-manager/commit/71e30a96)
- Don't deploy dummy services with CSI components [eb3ec796](https://github.com/longhorn/longhorn-manager/commit/eb3ec796)
- Add logs for volume attachment and detachment [86571643](https://github.com/longhorn/longhorn-manager/commit/86571643)
- Add metrics about disk conditions [d438b2c5](https://github.com/longhorn/longhorn-manager/commit/d438b2c5)
- Clean up orphaned snapshot attachment tickets while upgrading [c6261ead](https://github.com/longhorn/longhorn-manager/commit/c6261ead)
- Don't create an attachment ticket for a deleted snapshot [e7bf4f8e](https://github.com/longhorn/longhorn-manager/commit/e7bf4f8e)
- controller: IM controller updates unknown IM handling [2797d88e](https://github.com/longhorn/longhorn-manager/commit/2797d88e)
- controller: Node controller skips unknown IM cleanup [7dcf0396](https://github.com/longhorn/longhorn-manager/commit/7dcf0396)
- refactor: cleanup imports [aafaf525](https://github.com/longhorn/longhorn-manager/commit/aafaf525)
- Clean up e.spec.upgradedReplicaAddressMap when live upgrade is no longer needed [c8ad5257](https://github.com/longhorn/longhorn-manager/commit/c8ad5257)
- Waiting for old Longhorn manager to be fully removed before starting upgrade path [cd77ecd2](https://github.com/longhorn/longhorn-manager/commit/cd77ecd2)
- Make webhook deletion foreground to make sure old webhook pods are deleted before new Longhorn manager start upgrading [b69977b4](https://github.com/longhorn/longhorn-manager/commit/b69977b4)
- Improve log level for reconcilidation error [74128f5c](https://github.com/longhorn/longhorn-manager/commit/74128f5c)
- vendor: update dependencies [06605c89](https://github.com/longhorn/longhorn-manager/commit/06605c89)
- Revert #5017 [d69bd212](https://github.com/longhorn/longhorn-manager/commit/d69bd212)
- Fix typo [e83804ac](https://github.com/longhorn/longhorn-manager/commit/e83804ac)
- vendor: update dependencies [2b96f2cd](https://github.com/longhorn/longhorn-manager/commit/2b96f2cd)
- share_manager_controller: unexport and unmount volume before detaching it [14f50a82](https://github.com/longhorn/longhorn-manager/commit/14f50a82)
- share_manager_controller: avoid flooding messages [0c6b7d5f](https://github.com/longhorn/longhorn-manager/commit/0c6b7d5f)
- fix(restore): dr volume failed by delete lock [e5055c3f](https://github.com/longhorn/longhorn-manager/commit/e5055c3f)
- feat(volume): add condition for wait backing image [c53e26a8](https://github.com/longhorn/longhorn-manager/commit/c53e26a8)
- fix(backup): ensure volume attached before taking backup [2eaae1f0](https://github.com/longhorn/longhorn-manager/commit/2eaae1f0)
- NIT: VolumeController: don't overwrite err by lastErr [540b7932](https://github.com/longhorn/longhorn-manager/commit/540b7932)
- Unit test: make the new engine name deterministic based on the existing engine [4ba047e8](https://github.com/longhorn/longhorn-manager/commit/4ba047e8)
- Make the new engine name deterministic based on the existing engine [5b5c8967](https://github.com/longhorn/longhorn-manager/commit/5b5c8967)
- vendor: fix CVE-2022-43756 [ae01550b](https://github.com/longhorn/longhorn-manager/commit/ae01550b)
- Refuse to create a duplicate engine [5b9508ca](https://github.com/longhorn/longhorn-manager/commit/5b9508ca)
- rwx volume: use bigger timeo, retrans and softerr for nfs client [15cacf77](https://github.com/longhorn/longhorn-manager/commit/15cacf77)
- rwx volume: use soft mode with long timeout for nfs client [2a3a2e83](https://github.com/longhorn/longhorn-manager/commit/2a3a2e83)
- offline rebuilding: check spc.backendStoreDriver using a defensive way [fdd7e206](https://github.com/longhorn/longhorn-manager/commit/fdd7e206)
- refactor(logging): change scheduling related message to debug [f86e6dc4](https://github.com/longhorn/longhorn-manager/commit/f86e6dc4)
- refactor: replace controller run abbreviations [8c9a180f](https://github.com/longhorn/longhorn-manager/commit/8c9a180f)
- Fix bug two active engine when volume migrating [95de656d](https://github.com/longhorn/longhorn-manager/commit/95de656d)
- refactor: replace resource informer abbreviations [7bee1783](https://github.com/longhorn/longhorn-manager/commit/7bee1783)
- refactor: replace resource lister abbreviations [db645aeb](https://github.com/longhorn/longhorn-manager/commit/db645aeb)
- refactor: fix typo [27cc3658](https://github.com/longhorn/longhorn-manager/commit/27cc3658)
- Refuse to delete known settings [25d686e3](https://github.com/longhorn/longhorn-manager/commit/25d686e3)
- backup store: support SMB/CIFS hidden share [65ec4d18](https://github.com/longhorn/longhorn-manager/commit/65ec4d18)
- Add replicaName argument to ReplicaRebuildVerify [15793672](https://github.com/longhorn/longhorn-manager/commit/15793672)
- Bump CurrentInstanceManagerAPIVersion [01543008](https://github.com/longhorn/longhorn-manager/commit/01543008)
- Fix SnapshotClone with engine identity validation enabled [7405549c](https://github.com/longhorn/longhorn-manager/commit/7405549c)
- Add validation flags to engine during live upgrade [18d02a6b](https://github.com/longhorn/longhorn-manager/commit/18d02a6b)
- Enable identity validation for deprecated EngineBinary flow [6fa5db88](https://github.com/longhorn/longhorn-manager/commit/6fa5db88)
- Use new identity validation flags and fields [57d03250](https://github.com/longhorn/longhorn-manager/commit/57d03250)
- Use versions of longhorn-engine and longhorn-instance-manager with engine identity flags and fields [13426037](https://github.com/longhorn/longhorn-manager/commit/13426037)
- refactor: cleanup imports [6b50ac8d](https://github.com/longhorn/longhorn-manager/commit/6b50ac8d)
- fix(support-bundle): missing informer in cacheSyncs [0f3e26bc](https://github.com/longhorn/longhorn-manager/commit/0f3e26bc)
- refactor: cleanup imports [e15bd1eb](https://github.com/longhorn/longhorn-manager/commit/e15bd1eb)
- Refactor CreateSnapshot to return error when backup cannot complete [c0f4cd92](https://github.com/longhorn/longhorn-manager/commit/c0f4cd92)
- feat(metrics): add pvc name to volume metrics [32a4b6b4](https://github.com/longhorn/longhorn-manager/commit/32a4b6b4)
- refactor: cleanup unused CopyHostDirectoryContent [23eab07e](https://github.com/longhorn/longhorn-manager/commit/23eab07e)
- feat(util): remove non-existing source directory log [3dd7ca30](https://github.com/longhorn/longhorn-manager/commit/3dd7ca30)
- refactor: duplicated package imported [12029f75](https://github.com/longhorn/longhorn-manager/commit/12029f75)
- refactor: cleanup imports [293ebe89](https://github.com/longhorn/longhorn-manager/commit/293ebe89)
- fix(recurring-job): failed with backup failed [e8cfdc82](https://github.com/longhorn/longhorn-manager/commit/e8cfdc82)
- fix(telemetric): collect SettingNameSnapshotDataIntegrityCronJob setting as boolean [c69f7bf6](https://github.com/longhorn/longhorn-manager/commit/c69f7bf6)
- Fix backing image manager webhook is never called [1153859e](https://github.com/longhorn/longhorn-manager/commit/1153859e)
- fix(backup): set backup error if backup target invalid [1bc5df5b](https://github.com/longhorn/longhorn-manager/commit/1bc5df5b)

#### longhorn-engine
- release: v1.5.2-rc1 [3da49587](https://github.com/longhorn/longhorn-engine/commit/3da49587)
- Fix backport conflicts. Make vendor/modules agree with go.mod [dcf4ec79](https://github.com/longhorn/longhorn-engine/commit/dcf4ec79)
- Update vendor after backupstore merge [b0f09344](https://github.com/longhorn/longhorn-engine/commit/b0f09344)
- feat(log): add function name and line number in log message [7ed95dd9](https://github.com/longhorn/longhorn-engine/commit/7ed95dd9)
- ci: pause integration testing for s390x [8d7c586d](https://github.com/longhorn/longhorn-engine/commit/8d7c586d)
- refactor: replace fmt.Errorf with errors.Wrapf [80dedf19](https://github.com/longhorn/longhorn-engine/commit/80dedf19)
- fix: rebuilding in volume.meta blocks engine start [3848c261](https://github.com/longhorn/longhorn-engine/commit/3848c261)
- Fix 6481 - unbounded growth in BackupList from failure retries. [bb2c2f6e](https://github.com/longhorn/longhorn-engine/commit/bb2c2f6e)
- Fix 6481 - unbounded growth in BackupList from failure retries. [773d1973](https://github.com/longhorn/longhorn-engine/commit/773d1973)
- Add replica-instance-name argument to verify-rebuild-replica command [9475221d](https://github.com/longhorn/longhorn-engine/commit/9475221d)
- Change Name to VolumeName and cache VolumeName in controller client [33f28010](https://github.com/longhorn/longhorn-engine/commit/33f28010)
- Fix SnapshotClone with engine identity validation enabled [23b63b7d](https://github.com/longhorn/longhorn-engine/commit/23b63b7d)
- Bump ControllerAPIVersion [a45b3d49](https://github.com/longhorn/longhorn-engine/commit/a45b3d49)
- Polish identity validation server interceptor [b128fafd](https://github.com/longhorn/longhorn-engine/commit/b128fafd)
- Add identity verification integration tests for sync agent [3185dc9b](https://github.com/longhorn/longhorn-engine/commit/3185dc9b)
- Add identity verification to sync agent when acting as client and server [3e919743](https://github.com/longhorn/longhorn-engine/commit/3e919743)
- Prevent integration flakes due to closed socket [e74e2099](https://github.com/longhorn/longhorn-engine/commit/e74e2099)
- Remove replica --volume-name flag in favor of the new global one [4cf7559e](https://github.com/longhorn/longhorn-engine/commit/4cf7559e)
- Polish up engine identity log and error messages [2470a985](https://github.com/longhorn/longhorn-engine/commit/2470a985)
- Add identity validation integration tests [e2967448](https://github.com/longhorn/longhorn-engine/commit/e2967448)
- Add identity validation flags to CLI commands [dfbd8ead](https://github.com/longhorn/longhorn-engine/commit/dfbd8ead)
- Add instanceName field to ReplicaAddress message [1d14929d](https://github.com/longhorn/longhorn-engine/commit/1d14929d)
- Add gRPC identity interceptor to servers and clients [8e6e809d](https://github.com/longhorn/longhorn-engine/commit/8e6e809d)

#### longhorn-instance-manager
- release: v1.5.2-rc1 [061e041](https://github.com/longhorn/longhorn-instance-manager/commit/061e041)
- feat(log): add function name and line number in log message [0971e04](https://github.com/longhorn/longhorn-instance-manager/commit/0971e04)
- Error out when continuously receiving errors while watching instances [b14989a](https://github.com/longhorn/longhorn-instance-manager/commit/b14989a)
- vendor: update longhorn-spdk-engine [b2c88ac](https://github.com/longhorn/longhorn-instance-manager/commit/b2c88ac)
- vendor: update dependencies [4c049cd](https://github.com/longhorn/longhorn-instance-manager/commit/4c049cd)
- spdk: bind mount /host/sys to /sys [6d3105f](https://github.com/longhorn/longhorn-instance-manager/commit/6d3105f)
- Dockerfile: add nvme cli 2.5 [a522c90](https://github.com/longhorn/longhorn-instance-manager/commit/a522c90)
- command: correct the normal case when shutdown [be44b1d](https://github.com/longhorn/longhorn-instance-manager/commit/be44b1d)
- Fix SnapshotClone with engine identity validation enabled [67ae050](https://github.com/longhorn/longhorn-instance-manager/commit/67ae050)
- Differentiate between volumeName and backupVolumeName in BackupRestore [caebf64](https://github.com/longhorn/longhorn-instance-manager/commit/caebf64)
- Fix naming in SnapshotClone fields [130c574](https://github.com/longhorn/longhorn-instance-manager/commit/130c574)
- Add replicaName fields to messages and ensure they are used [3b735d9](https://github.com/longhorn/longhorn-instance-manager/commit/3b735d9)
- Add FromControllerName field to EngineSnapshotCloneRequest [b1863a2](https://github.com/longhorn/longhorn-instance-manager/commit/b1863a2)
- Use fields from ProxyEngineRequest to instantiate tasks and controller clients [9d6dd06](https://github.com/longhorn/longhorn-instance-manager/commit/9d6dd06)
- Add extra field to ProxyEngineRequest [1ef4db8](https://github.com/longhorn/longhorn-instance-manager/commit/1ef4db8)
- Use version of longhorn-engine with engine identity flags and fields [c925ae3](https://github.com/longhorn/longhorn-instance-manager/commit/c925ae3)
- nit: fix incorrect error log [861572f](https://github.com/longhorn/longhorn-instance-manager/commit/861572f)
- vendor: update go-spdk-helper [3a4177f](https://github.com/longhorn/longhorn-instance-manager/commit/3a4177f)

#### longhorn-share-manager
- release: v1.5.2-rc1 [cd85138](https://github.com/longhorn/longhorn-share-manager/commit/cd85138)
- feat(log): add function name and line number in log message [e3f14ae](https://github.com/longhorn/longhorn-share-manager/commit/e3f14ae)
- Create volume export config when volume is ready [f805aaa](https://github.com/longhorn/longhorn-share-manager/commit/f805aaa)
- vendor: add dependencies [8a167c3](https://github.com/longhorn/longhorn-share-manager/commit/8a167c3)
- script: support custom TAG [72060f5](https://github.com/longhorn/longhorn-share-manager/commit/72060f5)
- rpc: implement Unmount server and client functions [a86e599](https://github.com/longhorn/longhorn-share-manager/commit/a86e599)
- proto: add Unmount method [b11b102](https://github.com/longhorn/longhorn-share-manager/commit/b11b102)
- fix(share-manager): remount volume if mount point became ro [e106eab](https://github.com/longhorn/longhorn-share-manager/commit/e106eab)

#### backing-image-manager
- release: v1.5.2-rc1 [71d8452](https://github.com/longhorn/backing-image-manager/commit/71d8452)
- feat(log): add function name and line number in log message [86339af](https://github.com/longhorn/backing-image-manager/commit/86339af)

**Comment by innobead** [comment]
I just removed the below item, because we already have install and upgrade testing in https://github.com/longhorn/longhorn/wiki/Release-Test-Plan---1.5.2#e2e-pipelines. 

> ~- [ ] Run e2e regression for pre-GA milestones (`install`, `upgrade`) - @yangchiu~  

cc @longhorn/qa

**Comment by innobead** [comment]
@khushboo-rancher @yangchiu 

For the below tasks, you just need to mark them finished after we finish the items in the test plan https://github.com/longhorn/longhorn/wiki/Release-Test-Plan---1.5.2, so there is no need to do extra tests here by yourself. 

>     - [ ] Regression test plan (manual) - @khushboo-rancher (https://github.com/longhorn/longhorn/wiki/Release-Test-Plan---1.5.2)
>     - [ ] Run e2e regression for pre-GA milestones (`install`, `upgrade`) - @yangchiu

**Comment by innobead** [comment]
@longhorn/qa Please help use https://github.com/longhorn/longhorn/releases/tag/v1.5.2-rc2 for the following tests. 

### Change logs after RC1. 

#### longhorn-ui
- release: v1.5.2-rc2 [7de83dc](https://github.com/longhorn/longhorn-ui/commit/7de83dc)

#### longhorn-manager
- release: v1.5.2-rc2 [ed60399f](https://github.com/longhorn/longhorn-manager/commit/ed60399f)
- fix(backup): fix backup status if target becomes invalid during creating backup [0aa4a19c](https://github.com/longhorn/longhorn-manager/commit/0aa4a19c)
- fix(setting): checking annotation is nil or not [3af64f92](https://github.com/longhorn/longhorn-manager/commit/3af64f92)
- Don't crash when upgrading volume attachments after a previous failure [a4a8d314](https://github.com/longhorn/longhorn-manager/commit/a4a8d314)

#### longhorn-engine
- release: v1.5.2-rc2 [e74a7f05](https://github.com/longhorn/longhorn-engine/commit/e74a7f05)

#### longhorn-instance-manager
- release: v1.5.2-rc2 [c8e06f0](https://github.com/longhorn/longhorn-instance-manager/commit/c8e06f0)

#### longhorn-share-manager
- release: v1.5.2-rc2 [d307a45](https://github.com/longhorn/longhorn-share-manager/commit/d307a45)

#### backing-image-manager
- release: v1.5.2-rc2 [2c5b1a1](https://github.com/longhorn/backing-image-manager/commit/2c5b1a1)

**Comment by innobead** [comment]
https://github.com/longhorn/longhorn/tree/v1.5.2-rc3 is ready. 

### Change logs after RC2. 

#### longhorn-ui
- release: v1.5.2-rc3 [5859ee7](https://github.com/longhorn/longhorn-ui/commit/5859ee7)
- Duplicate MIME type "text/html" in /var/config/nginx/nginx.conf [94323f9](https://github.com/longhorn/longhorn-ui/commit/94323f9)

#### longhorn-manager
- release: v1.5.2-rc3 [4af769bb](https://github.com/longhorn/longhorn-manager/commit/4af769bb)
- fix(backup): backupInfo might be nil [5aa3a903](https://github.com/longhorn/longhorn-manager/commit/5aa3a903)
- fix(backup): secret annotations might be nil [00edb605](https://github.com/longhorn/longhorn-manager/commit/00edb605)

**Comment by roger-ryao** [comment]
Run security testing of container images for pre-GA milestones:

[v1.5.2-rc3](https://ci.longhorn.io/job/private/job/image-security-scan/1204/): 72 HIGH, 0 CRITICAL

**Comment by innobead** [comment]
@nitendra-suse you still have 2 manual testing. Do you need any helps?

@roger-ryao @chriscchien Please help @nitendra-suse if needed. We are targeting to release on Thai Friday. Thanks.

**Comment by nitendra-suse** [comment]
> @nitendra-suse you still have 2 manual testing. Do you need any helps? 
> I am about to finish [Kubernetes upgrade test](https://longhorn.github.io/longhorn-tests/manual/pre-release/upgrade/kubernetes-upgrade-test/) - Nitendra item , Roger is helping with other item

**Comment by roger-ryao** [comment]
> > @nitendra-suse you still have 2 manual testing. Do you need any helps?
> > I am about to finish [Kubernetes upgrade test](https://longhorn.github.io/longhorn-tests/manual/pre-release/upgrade/kubernetes-upgrade-test/) - Nitendra item , Roger is helping with other item

Hi @innobead 
   I've just completed the [Test backup listing S3/NFS](https://longhorn.github.io/longhorn-tests/manual/pre-release/stress/backup-listing/) item.

**Comment by innobead** [comment]
Awesome. Thanks @nitendra-suse @roger-ryao .

**Comment by innobead** [comment]
@chriscchien Please help check longhorn chart GA PR. https://github.com/longhorn/charts/pull/134
@yangchiu Please help verify longhorn GA PR. https://github.com/longhorn/longhorn/pull/7030

After all passed, we will release 1.5.2.

**Comment by yangchiu** [comment]
Run core testing (install, upgrade) for the GA build from the previous patch and the last patch of the previous feature release (1.5.1).

[amd64](https://ci.longhorn.io/job/public/job/v1.5.x/job/v1.5.x-longhorn-tests-sles-amd64/180/): 0 failure
[arm64](https://ci.longhorn.io/job/public/job/v1.5.x/job/v1.5.x-longhorn-tests-sles-arm64/179/): 0 failure
[amd64 upgrade](https://ci.longhorn.io/job/public/job/v1.5.x/job/v1.5.x-longhorn-upgrade-tests-sles-amd64/172/): 0 failure
[arm64 upgrade](https://ci.longhorn.io/job/public/job/v1.5.x/job/v1.5.x-longhorn-upgrade-tests-sles-arm64/172/): 0 failure

**Comment by innobead** [comment]
@chriscchien any updated about longhorn chart testing?

**Comment by chriscchien** [comment]
Verify longhorn chart PR to ensure all artifacts are ready for GA (install, upgrade)

- [x] https://github.com/longhorn/charts/pull/134#issuecomment-1792198616
- [x] https://github.com/longhorn/longhorn/pull/7030#issuecomment-1792199024

**Comment by innobead** [comment]
@PhanLe1010 @rebeccazzzz @khushboo-rancher Please help with the Rancher chart release and update the upgrade responder. 1.5.2 is the stable release now.

**Comment by dominch** [comment]
I just made upgrade from 1.5.1 to 1.5.2 and engine for all volumes to 1.5.2 
55/57 volumes without any issue, 2 stuck in some resize process (I did not resized anything for some time) 
Also I can't pause/cancel this process, oddly it happened to pv with 3 nodes mysql-cluster, two other nodes were ok. Second volume that has this issue is very small instance with few megabytes of usage. Should I dig into this problem or just redeploy those workloads?

**Comment by innobead** [comment]
@dominch Let's create a specific issue for the discussion.

**Comment by dominch** [comment]
Sure, creating it now,

**Comment by dominch** [comment]
Ok, just for reference its: #7053 
support bundle is attaching now from UI

**Comment by innobead** [comment]
1.5.2 rancher chat has been paused in favor of the upcoming 1.5.3, which will resolve a regression related to encryption volume.

cc @longhorn/qa
